modules/logging-release-notes-5-7-11.adoc

```diff
@@ -0,0 +1,13 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-11_{context}"]
+= Logging 5.7.11
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[Logging Bug Fix 5.7.11]
```

Comment:
Punctuation:
```suggestion
This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[Logging Bug Fix 5.7.11].
```
Also I notice the other entries say more fully "OpenShift Logging", so maybe:
```suggestion
This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[OpenShift Logging Bug Fix 5.7.11].
```

Reply:
Logging is a legal approved short name, OpenShift Logging is not the current legal name.